src/fermo/pages/pages_dashboard.py

```diff
@@ -969,6 +1010,97 @@ def call_feature_export_button():
             'width' : '60%'},
         )
 
+
+
```

Comment:
just to test the comment function: why are there so many empty lines here?

Reply:
No reason, could be removed. Linting and automatic formatting is not implemented yet but will be in a future version.